File: SWexpert/D1/2072/2072.java
import java.util.Scanner;
 
public class Solution{
    public static void main(String[] args){
        Scanner scan = new Scanner(System.in);
        int t;
        t = scan.nextInt();
        for(int i = 0; i < t; i++){
            int sum = 0;
            for(int j = 0; j < 10; j++){
                int p;
                p = scan.nextInt();
                if(p % 2 == 0)
                {
                }
                else
                    sum += p;
            }
            System.out.printf("#%d %d\n", i + 1, sum);
        }
    }
}

File: SWexpert/D1/2072/2072.cpp
#include<iostream>
using namespace std;

int main(void){
    int t;
    cin >> t;
    
    for(int i = 1; i <= t; i++)
    {
        int sum = 0;
        for(int j = 1; j <= 10; j++)
        {
            int p[10] = {};
            cin >> p[j-1];
            if((p[j-1] % 2) == 0)
            {
            }
            else
                sum += p[j-1];
        }
        cout << "#" << i << " " << sum << "\n";
    }
}
